grr/server/grr_response_server/gui/ui/lib/api/api_interfaces.ts

```diff
@@ -36,6 +36,17 @@ export declare interface ApiClientLabel {
   readonly name?: string;
 }
 
+export declare interface ApiClientsLabels {
```

Comment:
Please use the exact names from the Protobuf files (`ApiListClientsLabelsResult` see proto/grr_response_proto/api/client.proto). This allows us to switch from the manual definitions to auto-generated ones in the near future.

Reply:
Also, please add a one-line JSDoc. I know it doesn't really contain information, but it is a necessary evil to keep the linter happy until we have code generation in place.

Reply:
Got it